website/static/css/style.css

```diff
@@ -59,8 +59,8 @@ a {
     list-style-type: none;
 }
 
-/* Cookie Banner */
-#cookieBanner {
+/* Warning Banner for cookies and browser depreciation goes on bottom of page */
+.warningBanner {
```

Comment:
Wasn't specified in the ticket, but I'd make the IE warning banner a different color from the cookie warning banner since it's possible to see both simultaneously.   Maybe yellow with black text like the examples in the ticket - it's annoying and gets your attention and hopefully will steer them to another browser.  Maybe get product's input?

Reply:
I put a white border which separates the banners when stacked, product seemed ok with that and gave me official language.